casper/src/test/resources/RevAddressTest.rho

```diff
@@ -62,5 +63,13 @@ in {
       revAddress!("fromPublicKey", "F0F0".hexToBytes(), *retCh) |
       rhoSpec!("assert", (Nil, "== <-", *retCh), "correct RevAddress", *ackCh)
     }
+  } |
+
+  contract test_fromUnfName(rhoSpec, _, ackCh) = {
+    new unf1, unf2, r1, r2 in {
+      revAddress!("fromUnfName", *unf1, *r1) |
```

Comment:
we should use the full name, `fromUnforgeableName`

Reply:
Even better: `fromUnforgeable`. It's not that anyone can pass a name. We always send processes, and always receive names.

Reply:
Done